modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/Rule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+/**
+ * Interface for a generic rule with the standard fields relevant for all policy rules
+ */
+public interface Rule {
```

Comment:
at some places in the code (e.g. in `model`) we would write `IRule` for an interface

Reply:
I saw that, but this is actually a bad practice. An interface is a type and a type does not care whether it is an interface or not. Also for the user of the type, there is no real difference between an interface and a class, so this convention does not make sense for him. In the implementing code, this is also not helpful because it is easily visible that an interface is implemented not extended, so the I does not give any more information. Even the IDE shows nicely whether you have an interface, an enum, an exception or a class